doc/source/ci-cd/github-actions.rst

```diff
@@ -0,0 +1,50 @@
+==============
+GitHub-Actions
+==============
+
+Here is a minimal complete example workflow to create a Code Scanning action using Bandit by using GitHub Actions from PyCQA's `bandit-action <https://github.com/PyCQA/bandit-action>`.
+
+==============
+Example YAML code for GitHub Actions Pipeline
+==============
+
+.. code-block:: yaml
+
+    name: Bandit
+
+    on:
+        workflow_dispatch:
+
+    jobs:
+        analyze:
+        runs-on: ubuntu-latest
+        permissions:
+        # required for all workflows
+        security-events: write
+        # only required for workflows in private repositories
+        actions: read
+        contents: read
+        steps:
+        - name: Perform Bandit Analysis
+          uses: PyCQA/bandit-action@v1
+
+==============
+Inputs
+==============
```

Comment:
Same as https://github.com/PyCQA/bandit/pull/1172#discussion_r1886676691 but for this section title instead